src/index.html

```diff
@@ -3,12 +3,48 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Moyo</title>
+    <link href="https: //fonts.googleapis.com/css2? family= Montserrat:wght@400;500;600 & family= Roboto:ital,wght@1,300;1,400;1,500 & display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__link">
+        <img src="images/logo.png" alt="logo images"
+        class="logo__image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops.html" class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets.html" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets.html" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo.ntml" class="nav__link">Photo</a>
```

Comment:
There is a typo in the 'href' attribute for the Photo link. Change 'photo.ntml' to 'photo.html'.